### PR TITLE
Add support for cosmic series

### PR DESCRIPTION
**Pull request by fabricematrat**

### CODE REVIEW

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/charmstore/2184/

File: internal/series/series.go [hunk 1]
```diff
@@ -53,6 +53,7 @@ var Series = map[string]SeriesInfo{
 	"zesty":   {true, Ubuntu, false, 0},
 	"artful":  {true, Ubuntu, true, 1.105},
 	"bionic":  {true, Ubuntu, true, 1.15},
+	"cosmic":  {true, Ubuntu, true, 1.107},
```
Comment (rogpeppe) [inline internal/series/series.go]: do we really want to prioritise it this much already?

**Review by rogpeppe — COMMENTED** [review]
LGTM, thanks.

File: internal/series/series.go (hunk 1 above)
Comment (mhilton) [reply internal/series/series.go]: No, it should have a priority of 0 and a comment saying to update the priority when it's released.

File: internal/charmstore/store.go [hunk 2]
```diff
@@ -799,6 +799,7 @@ var seriesScore = map[string]int{
 	"yakkety": 7,
 	"zesty":   8,
 	"artful":  9,
+	"cosmic":  10,
```
Comment (mhilton) [inline internal/charmstore/store.go]: This priority should be 0 I think, no-one will be able to install these charms (although people don't really make charms for non-lts releases).

**Review by mhilton — APPROVED** [review]
LGTM with some changes in priority.

**Comment by fabricematrat** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/charmstore-merge

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/charmstore/2185/

**Pull request merged**